api/WebSocket.json

```diff
@@ -82,47 +71,40 @@
           "description": "<code>WebSocket()</code> constructor",
           "support": {
             "chrome": {
-              "version_added": true
+              "version_added": "4"
             },
             "chrome_android": {
-              "version_added": true
+              "version_added": "18"
             },
             "edge": {
               "version_added": "12"
             },
-            "firefox": [
-              {
-                "version_added": "7"
-              },
-              {
-                "version_added": "4",
-                "version_removed": "7",
-                "notes": "Parameter <code>protocols</code> not supported."
-              }
-            ],
+            "firefox": {
+              "version_added": "7"
```

Comment:
Can you duplicate the "Moz" prefix data here? We need to do that for constructors IMO since that's where the prefix is used.